src/components/atoms/DeviceVideo/DeviceVideo.scss

```diff
@@ -12,6 +12,7 @@ $max-display-size: 460px;
     @include square-size(100%, 100%);
     margin: 0;
     border-radius: $border-radius--lg;
+    object-fit: cover;
```

Comment:
Zooms up a non-square video stream (example 4:3 ratio) to be square size thus both sides not having extra empty space